dotcms-integration/pom.xml

```diff
@@ -291,43 +289,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
No longer need this block now sources and resources are in the standard location